core/src/main/java/org/openedx/core/AppDataConstants.kt

```diff
@@ -10,4 +10,6 @@ object AppDataConstants {
 
     const val VIDEO_FORMAT_M3U8 = ".m3u8"
     const val VIDEO_FORMAT_MP4 = ".mp4"
-}
\ No newline at end of file
+
+    const val DOWNLOADS_CONFIRMATION_SIZE = 1073741824L
```

Comment:
Could you please explain what this size means?

Reply:
this is equal to 1 GB, added comment for explanation